cmd/kubeadm/app/features/features.go

```diff
@@ -38,6 +38,8 @@ const (
 	EtcdLearnerMode = "EtcdLearnerMode"
 	// WaitForAllControlPlaneComponents is expected to be alpha in v1.30
 	WaitForAllControlPlaneComponents = "WaitForAllControlPlaneComponents"
+	// ControlPlaneKubeletLocalMode is expected to be in alpha in v1.31, beta in v1.32
+	ControlPlaneKubeletLocalMode = "ControlPlaneKubeletLocalMode"
```

Comment:
IIUC, we limit the scope to `kubeadm join` and kubelet bootstrap conf. 
- The name seems to have a larger scope than it really is.

Reply:
You are right, so maybe `ControlPLaneJoinKubeletLocalMode` might be a better fit?

Reply:
this is something we could have discussed on the KEP, personally i'm fine either way.
we do add a paragraph for each new FG here, https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#feature-gates so users can understand it better.

Reply:
> we do add a paragraph for each new FG here, https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#feature-gates so users can understand it better.

Yes. This can be covered by the doc and comments.

Reply:
revisiting +1 to use ControlPlaneKubeletLocalMode  as FG name, we won't have more than 1 FGs that would conflict and i think there will be enough clarity around ControlPlaneKubeletLocalMode.

Reply:
> revisiting +1 to use ControlPlaneKubeletLocalMode as FG name, we won't have more than 1 FGs that would conflict and i think there will be enough clarity around ControlPlaneKubeletLocalMode.

OK.